modin/pandas/test/test_groupby.py

```diff
@@ -371,7 +371,12 @@ def maybe_get_columns(df, by):
     eval_len(modin_groupby, pandas_groupby)
     eval_sum(modin_groupby, pandas_groupby)
     eval_ngroup(modin_groupby, pandas_groupby)
-    eval_general(modin_groupby, pandas_groupby, lambda df: df.nunique())
+    eval_general(
+        modin_groupby,
+        pandas_groupby,
+        lambda df: df.nunique(),
+        check_exception_type=None,
```

Comment:
pandas raises exception here because of a bug pandas-dev/pandas#36698, modin processes well and do not raises any exception.

`check exception type=None` means that we should not fail the test if pandas raise exception and modin don't

Reply:
added comment